hedera-node/src/main/java/com/hedera/services/fees/AwareHbarCentExchange.java

```diff
@@ -44,18 +44,24 @@ public ExchangeRate activeRate() {
 
 	@Override
 	public ExchangeRateSet activeRates() {
-		return rates;
+		return grpcRates;
```

Comment:
Complained by Codecov. Should add a unit test to cover this function.

Reply:
Added in https://github.com/hashgraph/hedera-services/pull/1525